packages/babel-plugin-transform-runtime/test/fixtures/use-options/corejs-useES6Modules/output.mjs

```diff
@@ -1,8 +1,8 @@
 import _Reflect$construct from "@babel/runtime-corejs3/core-js-stable/reflect/construct";
-import _classCallCheck from "@babel/runtime-corejs3/helpers/esm/classCallCheck";
-import _inherits from "@babel/runtime-corejs3/helpers/esm/inherits";
-import _possibleConstructorReturn from "@babel/runtime-corejs3/helpers/esm/possibleConstructorReturn";
-import _getPrototypeOf from "@babel/runtime-corejs3/helpers/esm/getPrototypeOf";
+import _classCallCheck from "@babel/runtime-corejs3/helpers/classCallCheck";
```

Comment:
is `useES6Modules` a thing (it's in the fixture path)? Seems like maybe this test is not relevant any longer?